File: Android-app/domain/src/main/java/uj/roomme/domain/product/ProductModel.kt
package uj.roomme.domain.product

import uj.roomme.domain.common.CommonCostModel

data class ProductModel(
    val id: Int,
    val authorId: Int,
    val authorName: String,
    val cost: CommonCostModel,
    val name: String,
    val description: String,
    val reason: String,
    val quantity: Int,
    val bought: Boolean
)

File: Backend/RoomMe/RoomMe.API/Models/ProductModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RoomMe.API.Models
{
    public class ProductModel
    {
        public int Id { get; set; }
        public int AuthorId { get; set; }
        public string AuthorName { get; set; }
        public CommonCostModel Cost { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Reason { get; set; }
        public int Quantity { get; set; }
        public bool Bought { get; set; }
    }
}
